src/components/issues/Card.tsx

```diff
@@ -0,0 +1,105 @@
+import { FC, useState } from "react";
+import styles from "@/components/issues/Card.module.scss";
+import { toast, ToastTypes } from "@/helperFunctions/toast";
+
+const TASK_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks/`;
+import fetch from "@/helperFunctions/fetch";
+import { IssueCardProps } from "@/interfaces/issueProps.type";
+const { SUCCESS, ERROR } = ToastTypes;
+
+const Card: FC<IssueCardProps> = ({ issue }) => {
+	const created = new Date(issue.created_at).toDateString();
+	const [taskExists, setTaskExists] = useState(issue.taskExists ?? false);
+	const [isLoading, setIsLoading] = useState(false);
+
+	const getIssueInfo = () => {
+		const issueInfo: any = {
+			status: issue.state,
+			id: issue.id,
+		};
+
+		if (issue.assignee) {
+			issueInfo.assignee = issue.assignee.login;
+		}
+
+		return issueInfo;
+	};
+
```

Comment:
Suggestion :
```
function getIssueInfo() {
	return {
		id: issue.id,
		status: issue.state,
		assignee: issue.assignee?.login ?? '',
	};
}
```
Since we're only interested in creating and returning an object with certain properties, we can skip the extra variable assignment. As for the `assignee` property, if it is acceptable to have it as `assignee: ""` when there is no assignee, then we can refactor the `if` block.
As for using `function` syntax instead of `arrow fn`, I'm not sure what's the code convention that is being followed in this project, but it can be a good idea to separate `util` fns like this from other handlers.

Reply:
> Since we're only interested in creating and returning an object with certain properties, we can skip the extra variable assignment. As for the assignee property, if it is acceptable to have it as assignee: "" when there is no assignee, then we can refactor the if block.

This has been chosen deliberately becasuse in the BE code, if there's an isuse assignee, we would fetch the RDS user information. To avoid fetching RDS user info for `""` assignee, used an `if` block here.

> As for using function syntax instead of arrow fn, I'm not sure what's the code convention that is being followed in this project, but it can be a good idea to separate util fns like this from other handlers.

Not sure about the code convention as well. I generally prefer ES6 syntax, hence arrow function. Since this util is only being used for one component, I did see the need a separate it in a different say `utils/`.